go.mod

```diff
@@ -1,97 +1,106 @@
 module github.com/openshift/cluster-api-actuator-pkg
 
-go 1.22.1
+go 1.23.0
 
-toolchain go1.22.7
+toolchain go1.23.5
 
 require (
 	github.com/aws/aws-sdk-go v1.55.5
-	github.com/golangci/golangci-lint v1.61.0
+	github.com/golangci/golangci-lint v1.63.4
 	github.com/google/uuid v1.6.0
-	github.com/onsi/ginkgo/v2 v2.20.2
-	github.com/onsi/gomega v1.34.2
-	github.com/openshift/api v0.0.0-20240924155631-232984653385
-	github.com/openshift/cluster-api-actuator-pkg/testutils v0.0.0-20241119145735-af0b63d8343b
-	github.com/openshift/cluster-autoscaler-operator v0.0.1-0.20240509123215-40cadf8a4729
-	github.com/openshift/library-go v0.0.0-20240919205913-c96b82b3762b
-	github.com/openshift/machine-api-operator v0.2.1-0.20240924183942-9c3e4a04009a
+	github.com/onsi/ginkgo/v2 v2.22.2
+	github.com/onsi/gomega v1.36.2
+	github.com/openshift/api v0.0.0-20250131155403-30a036067514
+	github.com/openshift/cluster-api-actuator-pkg/testutils v0.0.0-20250122171707-86066d47a264
+	github.com/openshift/cluster-autoscaler-operator v0.0.1-0.20241204142113-43631b045675
+	github.com/openshift/library-go v0.0.0-20250129210218-fe56c2cf5d70
+	github.com/openshift/machine-api-operator v0.2.1-0.20250122132849-b397e885dafb
 	github.com/tidwall/gjson v1.18.0
-	k8s.io/api v0.31.1
-	k8s.io/apimachinery v0.31.1
-	k8s.io/client-go v0.31.1
+	k8s.io/api v0.32.1
+	k8s.io/apimachinery v0.32.1
+	k8s.io/client-go v0.32.1
 	k8s.io/klog v1.0.0
-	k8s.io/utils v0.0.0-20240711033017-18e509b52bc8
-	sigs.k8s.io/cluster-api v1.8.4
-	sigs.k8s.io/cluster-api-provider-aws/v2 v2.6.1
+	k8s.io/utils v0.0.0-20241104100929-3ea5e8cea738
+	sigs.k8s.io/cluster-api v1.9.4
+	sigs.k8s.io/cluster-api-provider-aws/v2 v2.7.1
 	sigs.k8s.io/cluster-api-provider-azure v1.17.1
```

Comment:
nit: curious as to why azure and gcp aren't being bumped here

Reply:
It doesn't matter much as we only use APIs for those, ideally they should be the minor release that follows the twin CAPI minor release (in this case CAPI 1.9), which is not out for all the providers. I'll bump them again across repos when we have also CAPA and CAPG releases upstream that follow CAPI 1.9, but in the end it is not really important as it is API only stuff.

Reply:
Gotcha, thanks for explaining! :)